src/contents.ts

```diff
@@ -631,9 +633,7 @@ namespace Private {
    */
   export
   function b64DecodeUnicode(str: string): string {
```

Comment:
DecodeUTF8, not Unicode, right? UTF8 and Unicode are two different things.

Also, it probably makes sense to make the text decoder outside of the function, and just use it in the function, rather than instantiating a new one each time.

Reply:
Thanks, fixed.